### PR TITLE
Feature/descw 984 plugin theme deploy

### PR DESCRIPTION
**Pull request by kshapka-bcgov**

### CODE REVIEW

File: deployments/kustomize/README.md [hunk 1]
```diff
@@ -348,8 +348,30 @@ spec:
...
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+- github.com/bcgov/wordpress/deployments/base/plugins_themes
+# Update to your license plate in the dev|test|prod namespace.
+namespace: 123456-dev
 
+images:
+  - name: wordpress-plugins-themes-run
+    newName: image-registry.openshift-image-registry.svc:5000/23576a-tools/wordpress-plugins-themes-run
```
Comment (ShawnTurple) [inline deployments/kustomize/README.md:368]: should update the license plate here as well

**Review by ShawnTurple — COMMENTED** [review]
can you change the license plate for the image

File: deployments/kustomize/README.md (hunk 1 above)
Comment (kshapka-bcgov) [reply deployments/kustomize/README.md:368]: Thank you.

**Review by ShawnTurple — APPROVED** [review]

**Review by robrien-bcgov — APPROVED** [review]
looks good

**Pull request merged**